build_tools/mako/configuration.py

```diff
@@ -100,17 +100,23 @@ def get_pixel4_default_target_list(skipped_target=None, batch_config=None):
   if batch_config is None:
     batch_config = []
   targets = [
-      TargetInfo(name="vmla", mako_tag="vmla"),
-      TargetInfo(name="dylib-llvm-aot",
+      TargetInfo(driver="vmla",
+                 hal_target_backend="vmla",
+                 taskset="f0",
```

Comment:
to have a fair comparison between vmla and dylib-sync lets keep both running on the same core with `taskset=f0`

Reply:
I think they should all be `taskset=80`?